queue_services/entity-filer/src/entity_filer/filing_processors/consent_continuation_out.py

```diff
@@ -0,0 +1,38 @@
+# Copyright © 2021 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""File processing rules and actions for the court order filing."""
+from contextlib import suppress
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+from typing import Dict
+
+from legal_api.models import Business, Document, DocumentType, Filing
+
+from entity_filer.filing_meta import FilingMeta
+
+
+def process(business: Business, cco_filing: Filing, filing: Dict, filing_meta: FilingMeta):
+    """Render the court order filing into the business model objects."""
+    cco_filing.court_order_file_number = filing['courtOrder'].get('fileNumber')
+    cco_filing.court_order_effect_of_order = filing['courtOrder'].get('effectOfOrder')
```

Comment:
Court order is optional so, your code will need to handle this scenario where the filing has no `courtOrder` property.

Please reference [designs](https://preview.uxpin.com/f3e9c0da57601f193d2f0c6b1946134b2ddeb2fe#/pages/160490823/comment/sitemap) and [schema](https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/schemas/consent_continuation_out.json)/[validation](https://github.com/bcgov/lear/blob/acb61ef0c437dc448a4793ff603b61e172180faa/legal-api/src/legal_api/services/filings/validations/consent_continuation_out.py#L38-L42) code when updating filer code.